docs/2_4_common_co-simulation.adoc

```diff
@@ -5,7 +5,7 @@
 Co-simulation exploits the modular structure of coupled problems in all stages of the simulation process beginning with the separate model setup and preprocessing for the individual subsystems in different simulation tools (which can be powerful simulators as well as simple C programs).
 During time integration, the simulation is again performed independently for all subsystems restricting the data exchange between subsystems to discrete communication points latexmath:[tc_i].
 For simulator coupling, also the visualization and post-processing of simulation data is done individually for each subsystem in its own native simulation tool.
-In different contexts, the communication points latexmath:[tc_i], the communication steps latexmath:[tc_i \rightarrow tc_{i+1}] and the communication step sizes latexmath:[hc_i := tc_{i+1} - tc_i] are also known as sampling points (synchronization points), macro steps and sampling rates, respectively.
+In different contexts, the communication points latexmath:[tc_i], the communication steps latexmath:[tc_i \rightarrow tc_{\mathit{i+1}}] and the communication step sizes latexmath:[hc_i := tc_{\mathit{i+1}} - tc_i] are also known as sampling points (synchronization points), macro steps and sampling rates, respectively.
```

Comment:
`\mathit{i+1}` is incorrect. Should be only `i+1`. 
Mathit should be used one for words that are meant to denote the same mathematical concept.

Reply:
fixed